edx-platform/pearson-theme/lms/templates/courseware/course_about_sidebar_header.html

```diff
@@ -7,62 +7,124 @@
 from django.urls import reverse
 from django.conf import settings
 from six import text_type
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+%>
+<%
+social_share_itp = configuration_helpers.get_value('social_share_itp', ' ')
+course_about_twitter_account = configuration_helpers.get_value('course_about_twitter_account', settings.PLATFORM_TWITTER_ACCOUNT )
+course_about_facebook_link = configuration_helpers.get_value('course_about_facebook_link', settings.PLATFORM_FACEBOOK_ACCOUNT)
 %>
 
 <header>
   % if static.get_value('course_about_show_social_links', True):
-  <div class="social-sharing">
-    <div class="sharing-message">${_("Share with friends and family!")}</div>
-    ## TODO: this should probably be an overrideable block,
-    ##       or something allowing themes to do whatever they
-    ##       want here (and on this whole page, really).
-      <%
-        site_domain = static.get_value('site_domain', settings.SITE_NAME)
-        site_protocol = 'https' if settings.HTTPS == 'on' else 'http'
-        platform_name = static.get_platform_name()
+    % if social_share_itp:
```

Comment:
The only difference between these code blocks is the Twitter and Facebook links, we can remove social_share_itp and its code block and change the Twitter and Facebook links to check first in the site configuration and if they are not set then take the platform settings. @amalbas 
You can use this tool to check the difference: https://www.diffchecker.com/diff